DSharpPlus/Net/Rest/DiscordApiClient.cs

```diff
@@ -2873,7 +2873,7 @@ internal async Task<DiscordMessage> EditWebhookMessageAsync(ulong webhook_id, st
             {
                 Content = builder.Content,
                 Embeds = builder.Embeds,
-                Mentions = builder.Mentions,
+                Mentions = new DiscordMentions(builder.Mentions ?? Mentions.All, builder.Mentions?.Any() ?? false),
```

Comment:
`Mentions` should still be `null` if the builder's `Mentions` are `null`.
Not only would this otherwise be inconsistent with previous behavior, but editing a webhook without specifying mentions on the builder would now set it to have everyone in the message mentioned.
(Note the `NullValueHandling` on the payload property.)

Reply:
Truly a flashback of #1054.